pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -194,13 +196,15 @@ var mToF = []modelToFuncs{
 		model:       "raspberrypi.rpi.raspberrypi,4-model-b.brcm,bcm2711",
 		initFunc:    InitLedCmd,
 		displayFunc: ExecuteLedCmd,
-		arg:         "ACT",
+		// Try led0 first and if not found, ACT
+		arg: "led0,ACT",
```

Comment:
This commit is introducing a nice feature, which is the support of different possible names for the same LED. I won't change any LED name here. I think it's better to do it in the other commit, this one should focus only in the new feature. Also, ACT is the default now, so I would change the other: "ACT,led0".